### ISSUE TITLE
Add a "welcome" page

### ISSUE THREAD

**Issue by adamjrice** [BurningFlipside/VolunteerSystem#113]
When we direct someone to the FVS, if it's their first time there, they're probably wondering "what the hell am I looking at?"

We need to expand on the help text and perhaps use a cookie to determine whether they've been to the FVS before; if not, maybe display the help in a popover, redirect them to the help page, or add a prominent "first time here?" call to action button. Something. It's very unfriendly at first.

**Comment by pboyd04** [comment]
We pop up the wizard on the first login, I can nuke your FVS user to let you see that again if you like. I guess we can add something there?

As for not logged in. I'm not sure how much clearer I can make it...
![image](https://user-images.githubusercontent.com/594097/160249482-3788257e-f859-4300-a3bb-5a14d7ccf247.png)

**Comment by adamjrice** [comment]
Right. I kinda forgot about the wizard. Could make a link to the help doc the last step in the wizard. Also could expand the help doc with an explanation of the views and the download options. Also, the explanation of the color-coding has an error in it—green is colored blue.

**Comment by pboyd04** [comment]
Green is working fine for me... 
![image](https://user-images.githubusercontent.com/594097/160252404-3885d258-f94a-42f9-b92b-1362a8886d22.png)
As is blue:
![image](https://user-images.githubusercontent.com/594097/160252419-98d8dcc8-9c74-4472-b0ec-f49ff50a5308.png)
Can you show me where green isn't working right (the shift ID would be helpful as well (i.e. click on it))

**Comment by adamjrice** [comment]
<img width="459" alt="Screen Shot 2022-03-26 at 13 29 32" src="https://user-images.githubusercontent.com/15695982/160252572-9793f4ee-6091-4b81-b658-2b57db23ff44.png">

**Comment by pboyd04** [comment]
Ah that color coding, bug in both the html and the markdown, fixed
